sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -349,6 +350,11 @@ static int sof_card_late_probe(struct snd_soc_card *card)
 		i++;
 	}
 
+	if (sof_rt5682_quirk & SOF_MAX98373_SPEAKER_AMP_PRESENT) {
+		snd_soc_dapm_disable_pin(dapm, "Left Spk");
+		snd_soc_dapm_disable_pin(dapm, "Right Spk");
+		return snd_soc_dapm_sync(dapm);
```

Comment:
don't return here, you will skip the hdmi initialization.

Reply:
yeah..my bad, will fix this